Source/MARS.Client.Client.pas

```diff
@@ -11,7 +11,7 @@ interface
 
 uses
   SysUtils, Classes
-  , MARS.Core.JSON
+  , MARS.Core.JSON, MARS.Core.MediaType
   , MARS.Client.Utils, MARS.Core.Utils

```

Comment:
Can you format this that way:
```pascal
uses
  SysUtils, Classes
  , MARS.Core.JSON
  , MARS.Core.MediaType
  ,  MARS.Client.Utils
  , MARS.Core.Utils
  ;
```
We was discussing this here: https://github.com/andrea-magni/MARS/pull/32

Reply:
I fixed this, thanks @bogdanpolak